nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2241,6 +2270,442 @@ async fn certificate_delete(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+// RNI
+
+/// Create an address lot.
+#[endpoint {
+    method = POST,
+    path = "/v1/system/networking/address-lot",
+    tags = ["external-networking"],
+}]
+async fn networking_address_lot_create(
+    rqctx: RequestContext<Arc<ServerContext>>,
+    new_address_lot: TypedBody<params::AddressLotCreate>,
+) -> Result<HttpResponseCreated<AddressLotCreateResponse>, HttpError> {
+    let apictx = rqctx.context();
+    let nexus = &apictx.nexus;
+    let params = new_address_lot.into_inner();
```

Comment:
For the purposes of ensuring consistent instrumentation of handlers we want everything besides `apictx` to be moved into the handler. Just drop these down two lines.

Reply:
Done.